specification/communication/data-plane/Email/preview/2021-10-01-preview/CommunicationServicesEmail.json

```diff
@@ -3,7 +3,7 @@
   "info": {
     "title": "CommunicationEmailClient",
     "description": "Azure Communication Email Service",
-    "version": "2021-10-01-preview"
+    "version": "2023-15-01-preview"
```

Comment:
what do you mean? 2023-15-01 ?

Reply:
This is the new preview API version (2023-15-01 is the date). We hope to take these changes to GA after this preview is released.

Reply:
@apattath I think Weidong is pointing out that the month and days are flipped, for January this should be `2023-01-15-preview`

Reply:
I just realized it should have been 2023-01-15-preview (in yyyy-mm-dd-preview format). I will update this.